gridstatus/ercot.py

```diff
@@ -101,28 +103,37 @@ def get_fuel_mix(self, date, verbose=False):
 
     def get_load(self, date, verbose=False):
```

Comment:
now that this supports multiple days, it would make sense add the following decorator.

```python
@support_date_range("1D")
def get_load(self, date, verbose=False):
...
```

This allows the user to call the function with a start and and date, but the `1D` tells grid status to only call the underlying function with a frequency one day at a time. 

It also has handle for parsing date strings and objects, so a user can pass in something like `ercot.get_load("dec 1, 2022")`.

you can see an example on the `get_as_prices` function

Reply:
Makes sense!